### PR TITLE
feat(communications): add badge component

### PR DESCRIPTION
**Pull request by yuvarajuv**
image
image
image

### CODE REVIEW

**Comment by send2alamelu** [comment]
@yuvarajuv, can this left and right icon section similar to button

**Comment by yuvarajuv** [comment]
> @yuvarajuv, can this left and right icon section similar to button

@send2alamelu Added left and right icon section

File: react/src/Communications/Badge/Badge.js [hunk 1]
```diff
@@ -1,7 +1,30 @@
...
+    Cyan: 'bg-cyan-500',
+  };
 
-export default Badge
+  const badgeColor = statusToColor[status] || 'bg-gray-500';
+  const badgeClasses = overwriteClass || `inline-flex items-center px-3 py-1 text-white font-semibold text-sm rounded-full ${badgeColor}`;
+  
+
+  return (
+    <div className={badgeClasses}>
```
Comment (send2alamelu) [inline react/src/Communications/Badge/Badge.js:22]: can u replace BoxItem for div and span and use overwrite class ?

**Review by send2alamelu — CHANGES_REQUESTED** [review]

File: react-examples/stories/Communications/Badge.stories.js [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+export const Error = Template.bind({});
+Error.args = {
+  value: 'Error',
+  status: 'Error',
+};
+
+export const Warning = Template.bind({});
+Warning.args = {
+  value: 'Warning',
+  status: 'Warning',
```
Comment (send2alamelu) [inline react-examples/stories/Communications/Badge.stories.js:47]: status value should be in lower case

**Review by send2alamelu — CHANGES_REQUESTED** [review]

File: react-examples/stories/Communications/Badge.stories.js [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+};
+
+export const LeftIcon = Template.bind({});
+LeftIcon.args = {
+  value: 'User',
+  status: 'Success',
+  leftIcon: <TwixtIcon type="user" variant="filled" size="16" color="color" />,
```
Comment (send2alamelu) [inline react-examples/stories/Communications/Badge.stories.js:54]: color should not be color

**Comment by send2alamelu** [comment]
@yuvarajuv, can u move the current badge and its stories to 

_deprecated directory

and change the stories as

stories\Badges.stories.js

```
title: 'Common/TwixtBadges',
component: TwixtBadges_DEPRECATED,
```

**Review by send2alamelu — APPROVED** [review]
LGTM

**Pull request merged**